src/main/java/chess/dao/CommandDao.java

```diff
@@ -0,0 +1,74 @@
+package chess.dao;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CommandDao {
+
+    private static final String URL = "jdbc:mysql://localhost:3306/chess";
+    private static final String USER = "user";
+    private static final String PASSWORD = "password";
+
+    public Connection getConnection() {
+        loadDriver();
+        Connection connection;
+        try {
+            connection = DriverManager.getConnection(URL, USER, PASSWORD);
+        } catch (final SQLException e) {
+            e.printStackTrace();
+            throw new RuntimeException("프로그램 실행 중 오류가 발생하였습니다. (SQL connection error)");
+        }
+        return connection;
+    }
```

Comment:
```suggestion
        loadDriver();
        try {
           return DriverManager.getConnection(URL, USER, PASSWORD);
        } catch (final SQLException e) {
            e.printStackTrace();
            throw new RuntimeException("프로그램 실행 중 오류가 발생하였습니다. (SQL connection error)");
        }
    }
```

이렇게 변경하면 connection의 null할당은 피할 수 있겠군요~